### PR TITLE
Announce Bazel 0.13

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: _posts/2018-05-02-bazel-0.13.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: posts
+title: "Bazel 0.13"
+---
+
+Bazel 0.13](https://github.com/bazelbuild/bazel/releases/tag/0.13.0) has just
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: missing `[`

File: _posts/2018-05-02-bazel-0.13.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## BUILD / bzl files
+
+
+
+*   The flag `--incompatible_show_all_print_messages` is removed. Messages generated by [print](https://docs.bazel.build/versions/master/skylark/lib/globals.html#print) statements from any package will be displayed as debug messages.
+*   The `--incompatible_disallow_uncalled_set_constructor` flag is no longer available, the set constructor has been completely removed. Use [depset](https://docs.bazel.build/versions/master/skylark/lib/globals.html#depset) instead.
+*   Variables `PACKAGE_NAME` and `REPOSITORY_NAME` are deprecated in favor of functions [package_name()](https://docs.bazel.build/versions/master/skylark/lib/native.html#package_name) and [repository_name()](https://docs.bazel.build/versions/master/skylark/lib/native.html#package_group).
+*   Do not auto-detect base in the [int()](https://docs.bazel.build/versions/master/skylark/lib/globals.html#int) function if the input starts with '0'.
+*   Strings have a new [.elems()](https://docs.bazel.build/versions/master/skylark/lib/string.html#elems) method, that provides an iterator on the characters of the string.
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: s/method,/method/

File: _posts/2018-05-02-bazel-0.13.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+## Android
+
+
+
+*   The android.jar target `//tools/defaults:android_jar` has been moved to `@bazel_tools//tools/android:android_jar`.
+*   In [android_binary](https://docs.bazel.build/versions/master/be/android.html#android_binary), the attribute `manifest_merger` has been removed, and the `legacy` merging strategy is no longer supported. Manifests are now merged according to the semantics on the [Android Studio documentation page](https://developer.android.com/studio/build/manifest-merge.html).
+*   The AAR output of an `android_library` target now packages the merged proguard.txt from the `proguard_specs` attribute. It does not contain the proguard specs of transitive dependencies.android_library AAR output now contains proguard.txt
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: \`proguard.txt\`

Comment (jin) [reply _posts/2018-05-02-bazel-0.13.md]: space after "dependencies.`

Comment (jin) [reply _posts/2018-05-02-bazel-0.13.md]: period after \`proguard.txt\`.

File: _posts/2018-05-02-bazel-0.13.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+*   Enable dependency checking for aar_import targets.
+
+
+## C++
+
+
+
+*   Remove `optional_*_flag` fields from CROSSTOOL, they are not used, and could be expressed using features.
+*   Unified action_configs for static libraries. So instead of 4 action_configs you only have to specify one - c++-link-static-library.
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: you only have to specify one \`c++-link-static-library\`?

File: _posts/2018-05-02-bazel-0.13.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+*   Properly handle tree artifacts on the link command line coming from a cc_library dependency. See [example](https://stackoverflow.com/questions/48417712/how-to-build-static-library-from-the-generated-source-files-using-bazel-build/48524539#48524539) of usage. Generated header propagation is not yet supported.
+*   Windows: You can now pass `--experimental_shortened_obj_file_path=true` to shorten the object file path. Use this flag to mitigate C++ compilation errors from too long paths on Windows.
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: s/Windows:/In Windows,/

File: _posts/2018-05-02-bazel-0.13.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    1.  [setup_vc_env_vars(repository_ctx, vc_path)](https://github.com/bazelbuild/bazel/blob/9566f677a1093e3a3c0ddaed3f9ab34dd98e5e26/tools/cpp/windows_cc_configure.bzl#L184)
+    1.  [find_msvc_tool(repository_ctx, vc_path, tool)](https://github.com/bazelbuild/bazel/blob/9566f677a1093e3a3c0ddaed3f9ab34dd98e5e26/tools/cpp/windows_cc_configure.bzl#L202)
+*   Move (c/cxx)opts from legacy_compile_flags to user_compile_flags build variable.
+
+
+## Objective C
+
+
+
+*   The 'j2objc' configuration fragment has been exposed.
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: Link to https://docs.bazel.build/versions/master/skylark/lib/j2objc.html?

File: _posts/2018-05-02-bazel-0.13.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+*   Introduce `--incompatible_disable_objc_provider_resources` to turn off all resource-related fields of the Objc provider.
+
+
+## Misc
+
+
+
+*   In the Debian and Fedora packages, the global [bazelrc](https://docs.bazel.build/versions/master/user-manual.html#bazelrc) (typically installed at /etc/bazel.bazelrc) is an empty file, removing the previous `[--action_env](https://docs.bazel.build/versions/master/command-line-reference.html#flag--action_env)` and `[--test_env](https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_env)` defaults. If the file has been changed on your local system, your package manager may report that you need to manually merge the change.
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: \`/etc/bazel.bazelrc\`

File: _posts/2018-05-02-bazel-0.13.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+*   The BUILD_TIMESTAMP field in `volatile-status.txt` now contains seconds (and not milliseconds) since the epoch. For more information, see the [User's Manual](https://docs.bazel.build/versions/master/user-manual.html#workspace_status).
```
Comment (jin) [inline _posts/2018-05-02-bazel-0.13.md]: \`BUILD_TIMESTAMP\`

**Comment by laurentlb** [comment]
Thanks!
Done.

**Comment by laurentlb** [comment]
"Changes requested" -> how do I mark them as done?
Seems like I can't merge because of that.

**Pull request merged**